contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -492,15 +498,14 @@
       otherSourcePlaceholder: 'Please describe',
 
       // Privacy policy + terms of service
-      viewToSLink: 'View terms of service',
-      ToSCheck: 'I have read and agree to the terms of service',
+      viewToSLink: 'View Terms of Service',
       ToSRequiredMessage: 'Please accept our terms of service',
 
-      viewPrivacyPolicyLink: 'View privacy policy',
-      privacyPolicyCheck: 'I have read and agree to the privacy policy',
-      privacyPolicyRequiredMessage: 'Please accept our privacy policy',
-      contactMessage: 'Questions or concerns? Please email us at content@learningequality.org',
+      viewPrivacyPolicyLink: 'View Privacy Policy',
+      contactMessage: 'Questions or concerns? Please email us at ',
```

Comment:
In the designs, I overlooked that this string had the email address as a link at the end of it. Splitting a UI string into parts like this doesn't play nice with translating them, since the semantic structure may change when translated into another language. So the location of "content@learningequality.org" may not be in the same place, at the end in this case, with other languages. 

There might be a way to tokenize it but I'm not sure. I'd have to look in Kolibri to see if there are any examples of something like that, but generally we try to avoid hyperlinking partial portions of UI strings. For now, I think it's probably best to revert the `contactMessage` back to it's original form.